src/queryResult/queryResultWebViewController.ts

```diff
@@ -259,6 +258,19 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
         return res;
     }
 
+    public addResultSetSummary(
+        uri: string,
+        resultSetSummary: qr.ResultSetSummary,
+    ) {
+        let state = this.getQueryResultState(uri);
```

Comment:
does this need an early return in case there's no query result state for that uri?

Reply:
getQueryResultState will throw if no state is found for the uri